tests/installation/system_role.pm

```diff
@@ -68,7 +68,14 @@ sub assert_system_role {
 }
 
 sub run {
-    if (is_sle('<15') && !is_x86_64) {
+    if (is_sle('=12-sp5') && is_sles4sap) {
+        # Because is_sles4sap depends on the openQA settings and does not
+        # interogate the SUT to ask for the actual status, we need an extra
+        # needle check to make sure we are in the correct place and the System Role
+        # dialog did not come up.
+        assert_screen('partitioning-edit-proposal-button');
```

Comment:
There doesn't seem to be a verification run showing the `installation/system_role` test module being scheduled in 12-SP5. Seems to me that if the module is scheduled in 12-SP5 before `{{sles4sap_product_installation_mode_sle15}}`, then this `assert_screen` will fail as the test is not yet in the partitioning screen.

Reply:
The VRs are wrong I need to rerun them.